policy/modules/contrib/spamassassin.te

```diff
@@ -641,6 +641,9 @@ domain_read_all_domains_state(spamd_update_t)
 auth_use_nsswitch(spamd_update_t)
 auth_dontaudit_read_shadow(spamd_update_t)
 
+init_status(spamd_update_t)
+systemd_start_all_services(spamd_update_t)
```

Comment:
Instead of this, the spamassassin.service unit file should get a new private type so that sa-update is allowed to handle only this type.